docs/modules/ROOT/pages/access.adoc

```diff
@@ -56,37 +52,35 @@ To restrict a function's access to the owner only, add in the `assert_only_owner
 ----
 from openzeppelin.access.ownable.library import Ownable
 
-func protected_function{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }():
-    Ownable.assert_only_owner()
-    return ()
-end
+func protected_function{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() {
+    Ownable.assert_only_owner();
+    return ();
+}
 ----
 
 === Ownable library API
```

Comment:
Inconsistency with other pages that have an **API Specification** entry (like account, ERC20, ERC721...).

Reply:
Indeed. The alternative, if I'm not mistaken, would include adding the mock contract API entries (which I don't think add much value since they're mocks). I recommend we rectify these inconsistencies with a separate PR—this feels like #369. What do you think?

Reply:
I think that's fair.